src/components/NavigationBar/Icon.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withStyles from '@material-ui/core/styles/withStyles';
+import Button from '@material-ui/core/Button';
+import PersonIcon from '@material-ui/icons/Person';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+
+import NavBarDropDownMenu from './NavBarDropDownMenu';
+
+const Icon = ({
+  name,
+  isWarning,
+  handleClickIcon,
+  openIcon,
+  anchorEl,
+  handleClickAway,
+  menuHeader,
+  menuItems,
+  classes,
+}) => {
+  const iconOpened = `${classes.icons} + ${classes.offWhiteColor}`;
```

Comment:
I don't think the `+` in here actually does anything...if you take it out I think this still works.

Reply:
Didn't know that. Tested and it worked. Thanks.